### PR TITLE
Installation of new Eman version

### PR DESCRIPTION
**Pull request by DavidHerreros**
Modification of init.py with the steps required to install the new version of Eman based on Conda environments. Some minor changes were also added to template martching protocol and test to make them compatible with the new version.

### CODE REVIEW

**Comment by azazellochg** [comment]
3.0.0 can't be default version

**Comment by DavidHerreros** [comment]
Sorry, I make it the default for the testing and I forgot to change it, it should be fixed now

File: eman2/__init__.py [hunk 1]
```diff
@@ -142,3 +160,10 @@ def defineBinaries(cls, env):
                        tar='eman2.31.linux64.tgz',
                        commands=eman231_commands,
                        default=True)
+
+        env.addPackage('eman', version='3.0.0-alpha',
```
Comment (pconesa) [inline eman2/__init__.py:164]: Why not using the constant here?

Comment (pconesa) [reply eman2/__init__.py:164]: And I think the hyphen between 0 and alpha could be problemeatic....I'd remove it: 3.0.0.alpha?

**Comment by azazellochg** [comment]
Im getting this:
```
[azazello@xps13-9350 scipion-em-eman2]$ scipion installp -h
Scipion v3.0 () devel

WARNING: Target empty for command source ~/rc/conda.rc&&conda create -n eman3.0.0-alpha eman-deps-dev=22.1 -c cryoem -c defaults -c conda-forge && cd .. && mv eman2-* eman-source && mkdir eman-build && conda activate eman3.0.0-alpha && cd eman-build && cmake ../eman-source/ -DENABLE_OPTIMIZE_MACHINE=ON && make -j 4 && make install
```

**Comment by DavidHerreros** [comment]
> Im getting this:
> 
> ```
> [azazello@xps13-9350 scipion-em-eman2]$ scipion installp -h
> Scipion v3.0 () devel
> 
> WARNING: Target empty for command source ~/rc/conda.rc&&conda create -n eman3.0.0-alpha eman-deps-dev=22.1 -c cryoem -c defaults -c conda-forge && cd .. && mv eman2-* eman-source && mkdir eman-build && conda activate eman3.0.0-alpha && cd eman-build && cmake ../eman-source/ -DENABLE_OPTIMIZE_MACHINE=ON && make -j 4 && make install
> ```

I have fixed the target of the installation. Now the warning should not appear.

**Pull request merged**